package/src/components/Message/MessageSimple/MessageStatus.tsx

```diff
@@ -48,7 +48,7 @@ const MessageStatusWithContext = <
     },
   } = useTheme();
 
-  if (message.status === MessageStatusTypes.SENDING) {
+  if (message.status === MessageStatusTypes.SENDING || message.type === 'error') {
```

Comment:
should blocked message also have time icon like sending? doesnt make sense to me

Reply:
Fixed as per discussion by Fra 😄